Update to 3 in STEP and README.md (#2)
fatdogthebest/skills-review-pull-requests: README.md

</header>

<!--
  <<< Author notes: Step 5 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

## Step 5: Apply suggested changes

_Nicely done suggesting changes! :partying_face:_

> Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

Author: ```suggestion
Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).
Let me compleye thios
```

</header>

<!--
  <<< Author notes: Step 4 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

## Step 4: Suggest changes

_Nice work reviewing that pull request :sparkles:_

Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.

**What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.

### :keyboard: Activity: Suggest changes

> 1. On the pull request, click **Files changed**.

Author: ```suggestion
1. On the pull request, click **Yes**. Yes
```

> 1. Find the `index.html` changes.

Author: Yes
```suggestion
1. Find the `iYes
```

> 1. Find the `index.html` changes.

Author: ```suggestion
1. Find the `index.html` changes.
Test Test
```

> 1. Find the `index.html` changes.

Author: s

> 1. Find the `index.html` changes.

Author: ```suggestion
1. Find the `index.html` changes.s
```
